x-pack/plugins/apm/server/lib/transaction_groups/__snapshots__/queries.test.ts.snap

```diff
@@ -240,12 +240,8 @@ Array [
         "bool": Object {
           "filter": Array [
             Object {
-              "range": Object {
```

Comment:
The changes here are only a reflection in the changes of ordering the filters for readability.